src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  padding: 0;
+  margin-top: 20px;
+
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
It's good practice to provide fallback fonts in case the primary font fails to load. You've done this correctly by including Arial, Helvetica, and sans-serif as fallbacks.